src/RemoteTech/RemoteTech.csproj

```diff
@@ -131,46 +155,13 @@
     <Compile Include="RangeModel\AbstractRangeModel.cs" />
     <Compile Include="NetworkFeedback.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Resources\mark.png" />
-    <EmbeddedResource Include="Resources\texAll.png" />
-    <EmbeddedResource Include="Resources\texBackground.png" />
-    <EmbeddedResource Include="Resources\texButton.png" />
-    <EmbeddedResource Include="Resources\texButtonGray.png" />
-    <EmbeddedResource Include="Resources\texButtonGreen.png" />
-    <EmbeddedResource Include="Resources\texButtonRed.png" />
-    <EmbeddedResource Include="Resources\texButtonYellow.png" />
-    <EmbeddedResource Include="Resources\texDish.png" />
-    <EmbeddedResource Include="Resources\texEmpty.png" />
-    <EmbeddedResource Include="Resources\texFlightGreen.png" />
-    <EmbeddedResource Include="Resources\texFlightGreenDown.png" />
-    <EmbeddedResource Include="Resources\texFlightGreenOver.png" />
-    <EmbeddedResource Include="Resources\texFlightRed.png" />
-    <EmbeddedResource Include="Resources\texFlightRedDown.png" />
-    <EmbeddedResource Include="Resources\texFlightRedOver.png" />
-    <EmbeddedResource Include="Resources\texFlightYellow.png" />
-    <EmbeddedResource Include="Resources\texFlightYellowDown.png" />
-    <EmbeddedResource Include="Resources\texFlightYellowOver.png" />
-    <EmbeddedResource Include="Resources\texKnowledgeActive.png" />
-    <EmbeddedResource Include="Resources\texKnowledgeHover.png" />
-    <EmbeddedResource Include="Resources\texKnowledgeNormal.png" />
-    <EmbeddedResource Include="Resources\texOmni.png" />
-    <EmbeddedResource Include="Resources\texOmniDish.png" />
-    <EmbeddedResource Include="Resources\texPath.png" />
-    <EmbeddedResource Include="Resources\texPlanet.png" />
-    <EmbeddedResource Include="Resources\texSatellite.png" />
-    <EmbeddedResource Include="Resources\TimeQuadrantFcStatus.png" />
-    <EmbeddedResource Include="Resources\texNoCone.png" />
-    <EmbeddedResource Include="Resources\texNoOmniDish.png" />
-    <EmbeddedResource Include="Resources\texNoPath.png" />
-    <EmbeddedResource Include="Resources\texCone.png" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Resources\texBackground_left.png" />
-  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
+  <PropertyGroup>
+    <PostBuildEvent>
```

Comment:
It's good that you've removed the event itself. Those lines look like they're not needed anymore.